packages/amplify_auth_cognito/example/pubspec.yaml

```diff
@@ -36,10 +36,6 @@ dev_dependencies:
 # For information on the generic Dart part of this file, see the
 # following page: https://dart.dev/tools/pub/pubspec
 
-# Needed to avoid conflict for integration tests until flutter driver has null safe version of crypto https://github.com/flutter/flutter/issues/77282
```

Comment:
This was fixed with flutter 2.2.0, hence making it the min flutter version and removing this override.